api/v1alpha1/clustergroupupgrade_types.go

```diff
@@ -61,6 +61,9 @@ type ClusterGroupUpgradeSpec struct {
 	// Important: Run "make" to regenerate code after modifying this file
 	Clusters            []string                 `json:"clusters,omitempty"`
 	RemediationStrategy *RemediationStrategySpec `json:"remediationStrategy,omitempty"`
+	ManagedPolicy       []string                 `json:"managedPolicy,omitempty"`
+	//+kubebuilder:default=false
+	Enable bool `json:"enable,omitempty"`
```

Comment:
I will defer to standard practices for CRDs, but it seems like "enable" is a field that always has a value and should always be represented in the output. Should we remove the omitempty from this field?

Reply:
Right now all the members are using "omitempty" even if they are always used, apparently, but I agree with what you're saying. My proposal is to have a cleanup ticket to address these types of issue + any cleanup in the controller code once the restructuring functionality is in place.